src/openrct2-ui/windows/Window.h

```diff
@@ -95,6 +95,9 @@ rct_window* window_loadsave_open(int32_t type, const char* defaultName, loadsave
 rct_window* window_track_place_open(const struct track_design_file_ref* tdFileRef);
 rct_window* window_track_manage_open(struct track_design_file_ref* tdFileRef);
 
+void TrackPlaceClearProvisionalTemporarily();
```

Comment:
Is this really the best way to do it? We should eventually come up with a way to prevent peeps from accessing ghost elements. For the time being its okay, the way ghosts work should have a massive overhaul at some point.

Reply:
I think it is the best way at the moment. Its a real pain though as it means debug builds are too laggy for my computer with this constant placing. There are quite a few things that technically will cause a desync without doing something like this.